index.rst

```diff
@@ -0,0 +1,190 @@
+
+Doing Measurements:
+-------------------------
+
+The goal of the framework is to compare the results of a shape measurement algorithm with the known constant shear values which are produced with the Great3sims/GalSim simulation.  The algorithm must be housed in an lsst.meas.base measurement plugin so that it can be used by the LSST DM Stack.
+
+Our intial tests were with the CModel algorithm in lsst.meas.modelfit package, but this framework could be used to test any shape algorithm which is housed in a measurement plugin.
+
+Shear Bias Analysis:
+====================
+
+We wish to estimate the multiplicative and additive biases of our algorithm for both components of shear. For each shear component, this is done by plotting the measured shear against the applied shear from the epic catalog.  The multiplicative bias is the slope of our regression line, and the additive bias is the intercept.
+
+  .. figure:: /_static/figure_1.png
+     :name: figure_1
+     :target: _images/figure_1.png
+
+     Graph of regression line fit to 200 subfield measurements
+
+In this example, the regression lines lie very close to each other, with the measured shear multiplicatively biased by about +50%.  The intercepts deviate from zero by 7e-4 and 2e-4 respectively.
+
+Comparison of Shear Bias for Different Algorthms:
+-------------------------------------------------
+
+To see how different algorithms measure up against each other, we plot their shear biases against each other.  As in all of these tests, we compare against some measurement standard, and look for a significant change in the multiplicative or additive bias with other algorithms we are testing.
+
+This idea of using one of the algorithms as a measuring stick for the others originally started when we were testing algorithms which took a lot of computing time against “shortcuts”, which were clearly not as accurate but required less time.  For example, one of our tests was to compare measurements which used the full galaxy cutout (in this case 64x64) against measurements which used only a part of the cutout (24x24).  The goal in those test was to see whether a smaller cutout would change the bias, wth the full cutout as a measuring stick.
+
+  .. figure:: /_static/figure_2.png
+     :name: figure_2
+     :target: _images/figure_2.png
+
+     Comparison of shear bias with different stamp sizes
+
+One  question frequently asked is how this relates to the LSST requirements for achieving a particular accuracy in shear measurement.  The answer is that we have not attempted to calibrate the shear algorithms tested in this study, so our comparisons are only intended to show relative differences between different measurement algorithms.  A further calibration study is probably needed to say more. This study also did not include spatial variation in the Psf, which would be needed for an accurate assessment.
```

Comment:
You refer to the (plural) "shear algorithms tested in this study". Haven't you just used CModel? Why plural?